guides/common/modules/con_managing-red-hat-subscriptions.adoc

```diff
@@ -7,6 +7,11 @@ You must have a Red{nbsp}Hat subscription manifest containing a subscription all
 All subscription information is available in your Red Hat Customer Portal account.
 
 Before you can complete the tasks in this chapter, you must create a Red{nbsp}Hat subscription manifest in the Customer Portal.
+Only {Project} users on a disconnected network can create manifests on the Customer Portal.
+Only {Project} users on a connected network can create manifests using the Red{nbsp}Hat Hybrid Cloud Console.
+ifdef::satellite[]
+For more information on creating manifests on a connected network, see {RHDocsBaseURL}subscription_central/1-latest/html-single/creating_and_managing_manifests_for_a_connected_satellite_server/index[Creating and managing manifests for a connected {ProjectServer}] in _Subscription Central_.
+endif::[]
 
 ifdef::satellite[]
 To create, manage, and export a Red{nbsp}Hat subscription manifest in the Customer Portal, see https://access.redhat.com/documentation/en-us/subscription_central/2021/html/creating_and_managing_manifests_for_a_connected_satellite_server/index[Creating and managing manifests for a connected {ProjectServer}] in _Subscription Central_.
```

Comment:
Please consider moving it further below (before Additional resources?) or even a special module.

```suggestion
.Prerequisites
* Create a Red{nbsp}Hat subscription manifest.
** If your {Project} is connected, use the {RHCloud} to create the manifest.
ifdef::satellite[]
For more information, see {RHDocsBaseURL}subscription_central/1-latest/html-single/creating_and_managing_manifests_for_a_connected_satellite_server/index[Creating and managing manifests for a connected {ProjectServer}] in _Subscription Central_.
endif::[]
** If your {Project} is disconnected, use the Red Hat Customer Portal to create the manifest.
ifdef::satellite[]
For more information, see {RHDocsBaseURL}subscription_central/1-latest/html/getting_started_with_rhel_system_registration/adv-reg-rhel-using-rhsm_#using_manifests_con[Creating a subscription allocation for a disconnected Satellite Server] in _Subscription Central_.
```
Don't forget the `endif` on the next line :)